project_one.html

```diff
@@ -0,0 +1,199 @@
+            sign-ups required.
+          </p>
+          <div class="links">
+            <ul>
+              <li class="link">html</li>
+              <li class="link">css</li>
+              <li class="link">javaScript</li>
+            </ul>
+          </div>
+          <a href="#" class="pro"> See Project </a>
+        </section>
+      </div>
+
+      <br />
+      <br />
+      <div class="card">
+        <section class="snap">
+          <img src="./images/Snap3.png" alt="img" />
+        </section>
+        <section class="main">
+          <h1>Tonic</h1>
+          <div class="mid">
+            <span class="can"> CANOPY </span>
+            <img class="dot" src="./images/Counter.png" alt="" />
+            <span class="side"> BACK END DEV </span>
+            <img class="dot" src="./images/Counter.png" alt="" />
+            <span class="side"> 2015 </span>
+          </div>
+          <p class="para">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <div class="links">
+            <ul>
+              <li class="link">html</li>
+              <li class="link">css</li>
+              <li class="link">javaScript</li>
+            </ul>
+          </div>
+          <a href="#" class="pro"> See Project </a>
+        </section>
+      </div>
+
+      <br />
+      <br />
+
+      <div class="card">
+        <section class="snap">
+          <img src="./images/Snap4.png" alt="img" />
+        </section>
+        <section class="main">
+          <h1>
+            Multi-Post <br />
+            Stories
+          </h1>
+          <div class="mid">
+            <span class="can"> CANOPY </span>
+            <img class="dot" src="./images/Counter.png" alt="" />
+            <span class="side"> BACK END DEV </span>
+            <img class="dot" src="./images/Counter.png" alt="" />
+            <span class="side"> 2015 </span>
+          </div>
+          <p class="para">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <div class="links">
+            <ul>
+              <li class="link">html</li>
+              <li class="link">css</li>
+              <li class="link">javaScript</li>
+            </ul>
+          </div>
+          <a href="#" class="pro"> See Project </a>
+        </section>
+      </div>
+      <br />
+      <br />
+
+      <section class="about">
+        <h1 class="about_h">
+          About <br />
+          Myself
+        </h1>
+        <p class="para">
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hestiate to
+          contact me.
+        </p>
+        <a href="#" class="conn"> LET’S CONNECT </a>
+        <div class="icons">
+          <ul>
```

Comment:
- [ ] Please LET'S CONNECT is not intended to be a link it is a subtitle you can use hx element here since it a heading h1 - h6